ansible/roles/providers/tasks/vmware.yml

```diff
@@ -82,6 +82,24 @@
     group: root
     mode: 0644
 
+- name: Ensure VMware customisations are enabled
+  copy:
+    src: files/etc/cloud/cloud.cfg.d/10_vmware.cfg
+    dest: /etc/cloud/cloud.cfg.d/10_vmware.cfg
+    owner: root
+    group: root
+    mode: 0644
```

Comment:
What are the VMware customizations? Can you link to some documentation to make sure we want this across all OSes for vsphere?

Reply:
+1

Reply:
Oh drat. I meant to only do this for Ubuntu. Good catch, let me fix this up and add some documentation links.

Reply:
Turns out it isn't needed for the Ubuntu build - I was basing it off https://kb.vmware.com/s/article/59557 but according to https://williamlam.com/2022/06/using-the-new-vsphere-guest-os-customization-with-cloud-init-in-vsphere-7-0-update-3.html it is only used to switch to a new customization option available in vSphere >= 7.0u3. Leaving it out does not affect anything on Ubuntu 20.04 on my vSphere 7.0u3 environment, images still build and can be used to create clusters so I have removed it from the patch set. 

It might be something to consider in the future though, if the "traditional GOSC" customization method becomes deprecated.